apps/backend-sls/prisma/seed.ts

```diff
@@ -617,6 +619,9 @@ async function main(prisma: Prisma.PrismaClient) {
     },
     update: {},
   })
+
+  await prisma.$executeRaw`ALTER SEQUENCE "Question_id_seq" RESTART WITH 5`
```

Comment:
These statements are crucial and need to be kept up-to-date when extending the seed above. Otherwise the integration tests and application will fail, as the automatically created ids will start at 0, but that already exists due to the seeding. Will cause unique index errors.